engines/base/src/main/java/org/alfresco/transform/base/config/MTLSConfig.java

```diff
@@ -132,7 +140,10 @@ public SslContextBuilder nettySslContextBuilder() throws UnrecoverableKeyExcepti
             sslContextBuilder.keyManager(keyManagerFactory);
         }
 
-        if(isTruststoreConfigured())
+        if(hostNameVerificationDisabled)
+        {
+            sslContextBuilder.trustManager(InsecureTrustManagerFactory.INSTANCE);
```

Comment:
https://stackoverflow.com/questions/64190476/what-is-the-equivalent-of-setsslhostnameverifiernoophostnameverifier-instance
InsecureTrustManagerFactory doesn't restrict itself to hostnames only, it disregards all security.

Reply:
And can't you just add an if at the "createHttpClientWithSslContext" method then?